molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisWebAppConfig.java

```diff
@@ -255,6 +255,11 @@ public PropertiesMessageSource uiFormMessageSource() {
     return new PropertiesMessageSource("ui-form");
   }
 
+  @Bean
+  public PropertiesMessageSource coreUiMessageSource() {
+    return new PropertiesMessageSource("core-ui");
```

Comment:
Should the framework have thrown an error when a referenced messageSource was not found ?

Reply:
It falls back to the more technical message format then.